helix-core/src/main/java/org/apache/helix/util/HelixUtil.java

```diff
@@ -470,11 +471,26 @@ public static int getSystemPropertyAsInt(String propertyKey, int propertyDefault
   }
 
   /**
-   * Get the value of system property
+   * Get the boolean value of system property
    * @param propertyKey
    * @param propertyDefaultValue
    * @return
    */
+  public static boolean getSystemPropertyAsBoolean(String propertyKey, String propertyDefaultValue) {
```

Comment:
We don't have to create this method to be redundant.
Simply, `Boolean.getBoolean(propertyKey, propertyDefaultValue)` is already what is needed, right?

Reply:
+1. Unless you need some error handling in the function.

Reply:
removed.